.changeset/tricky-mails-retire.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-app-api': patch
```

Comment:
```suggestion
'@backstage/core-app-api': minor
```

This package is 1.x so new features are `minor` :)

Reply:
Thanks for the heads up, I'll get this fixed!